### PR TITLE
Fix "Can't Find" flow 

### PR DESCRIPTION
**Pull request by NEUDitao**
Can't Find flow is now happier

### CODE REVIEW

File: packages/app/components/Queue/StudentQueueList.tsx [hunk 1]
```diff
@@ -298,3 +319,30 @@ function QueueQuestions({ questions, studentQuestion }: QueueProps) {
...
+      footer={[
+        <Button key="leave" danger onClick={props.leaveQueue}>
+          Leave Queue
+        </Button>,
+        <Button type="primary" key="rejoin" onClick={props.rejoinQueue}>
+          Rejoin Queue
+        </Button>,
+      ]}
+      closable={false}
+      title="哎呀！！"
```
Comment (willstenzel) [inline packages/app/components/Queue/StudentQueueList.tsx:342]: I like the humor but I think it's getting to be that time where we should avoid adding random stuff so we don't risk missing removing something like this before the semester starts.

**Comment by willstenzel** [comment]
When I tried testing it on my local, when I click "Leave Queue" on the `CantFindModal` I got a popup asking if I wanted to delete my previous question. Is this happening for anyone else?

Screen Shot 2020-08-31 at 12 41 14 AM

**Comment by NEUDitao** [comment]
> When I tried testing it on my local, when I click "Leave Queue" on the `CantFindModal` I got a popup asking if I wanted to delete my previous question. Is this happening for anyone else?
> 
> Screen Shot 2020-08-31 at 12 41 14 AM

fixed in anothe rPR

File: packages/app/components/Queue/StudentQueueList.tsx (hunk 1 above)
Comment (NEUDitao) [reply packages/app/components/Queue/StudentQueueList.tsx]: 哎呀

**Pull request merged**